src/components/StakingHierarchy.js

```diff
@@ -0,0 +1,465 @@
+        <Content>
+          <p>
+            Solo staking on the Ethereum Beacon Chain is the{" "}
+            <Gold>gold standard</Gold> for staking. It is trustless, provides
+            full participation rewards, and improves the decentralization of the
+            network.
+          </p>
+          <p>
+            Those considering solo staking should have at least 32 ETH, hardware
+            connected to the internet ~24/7, and some technical know-how.
+          </p>
+          <p>
+            Over time, tooling has improved to help users get past some of the
+            technical hurdles, including options to bypass the command-line
+            altogether, if that's not your thing.
+          </p>
+          <p>
+            Check out some solo staking tools, then get started staking on the
+            Prater Testnet over on the{" "}
+            <Link to="https://prater.launchpad.ethereum.org">
+              Staking Launchpad
+            </Link>
+            .
+          </p>
+          <ButtonLink
+            to="/staking/solo/"
+            onClick={() => {
+              trackCustomEvent({
+                eventCategory: `StakingHierarchy`,
+                eventAction: `Clicked`,
+                eventName: "clicked solo staking",
+              })
+            }}
+          >
+            More on solo staking
+          </ButtonLink>
+        </Content>
+      </Section>
+      <Section number="2">
+        <Ether>
+          <StyledEtherSvg size="90%" />
+        </Ether>
+        <Line />
+        <Header>
+          <h2>Staking as a service</h2>
+          <Pills className="subtext">
+            <p>Your 32 ETH</p>
+            <p>Your validator keys</p>
+            <p>Entrusted node operation</p>
+          </Pills>
+        </Header>
+        <Glyph>
+          <SaasGlyph />
+        </Glyph>
+        <Content>
+          <p>
+            Running your own hardware is always recommended if possible for the
+            strength and health of the network, but we recognize this isn't
+            always reasonable for all users.
+          </p>
+          <p>
+            If you don't want or don't feel comfortable dealing with hardware,
+            but still want to stake your 32 ETH, staking-as-a-service options
+            allow you to delegate the hard part, while you earn native block
+            rewards.
+          </p>
+          <p>
+            These options usually walk you through creating a set of validator
+            credentials, uploading your signing keys to them, and depositing
+            your 32 ETH. This allows the service to validate on your behalf.
+          </p>
+          <p>
+            This method of staking requires a certain level of trust in the
+            provider. To limit counter-party risk, the keys to withdrawal your
+            ETH are usually kept in your possession.
+          </p>
+          <ButtonLink
+            onClick={() => {
+              trackCustomEvent({
```

Comment:
Are these custom event necessary? It's just clicking a link to navigate to a page.

Reply:
We wanted to get some more data on which buttons were being used the most
cc: @nloureiro @konopkja